common/src/main/java/com/wynntils/mc/mixin/plugin/WynntilsMixinPlugin.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.mixin.plugin;
```

Comment:
Maybe this is how it is supposed to work with plugins, but my gut reaction is that this should live in `core` instead, so all classes in `mc.mixin` are real, sponge mixin classes. I vaguely remember that there could be problems otherwise. And in any case, I think it sort of breaks a "rule" we have for our code base. It is mixin-related, yes, but it is not a mixin in itself.

Reply:
It won't break mixin but I can move it. Where, exactly?

Reply:
`com.wynntils.core.mod` is my suggestion.

Reply:
I see it as a "core stuff that is generally needed to support this code base as being a mod".